tapglue-android-sdk/tapglue-android-sdk.iml

```diff
@@ -96,12 +96,12 @@
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" exported="" name="okio-1.6.0" level="project" />
     <orderEntry type="library" exported="" name="gson-2.5" level="project" />
-    <orderEntry type="library" exported="" name="logging-interceptor-2.7.0" level="project" />
     <orderEntry type="library" exported="" scope="TEST" name="hamcrest-core-1.3" level="project" />
+    <orderEntry type="library" exported="" name="logging-interceptor-2.7.0" level="project" />
     <orderEntry type="library" exported="" name="retrofit-2.0.0-beta2" level="project" />
     <orderEntry type="library" exported="" scope="TEST" name="junit-4.12" level="project" />
-    <orderEntry type="library" exported="" name="okhttp-2.7.0" level="project" />
     <orderEntry type="library" exported="" name="support-annotations-23.1.1" level="project" />
+    <orderEntry type="library" exported="" name="okhttp-2.7.0" level="project" />
     <orderEntry type="library" exported="" name="converter-gson-2.0.0-beta2" level="project" />
   </component>
 </module>
```

Comment:
Yeah, will do a cleanup PR for this separately.

On Wed, Feb 3, 2016, 9:18 AM John Nilsen <notifications@github.com> wrote:

> In tapglue-android-sdk/tapglue-android-sdk.iml
> <https://github.com/tapglue/android_sdk/pull/32#discussion_r51690260>:
>
> >      <orderEntry type="library" exported="" name="support-annotations-23.1.1" level="project" />
> > +    <orderEntry type="library" exported="" name="okhttp-2.7.0" level="project" />
> >      <orderEntry type="library" exported="" name="converter-gson-2.0.0-beta2" level="project" />
> >    </component>
> >  </module>
>
> this one probably also for gitignore
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tapglue/android_sdk/pull/32/files#r51690260>.
>